src/runtime_src/core/tools/common/ReportAieShim.cpp

```diff
@@ -0,0 +1,338 @@
+/*
+  Copyright (C) 2020 Xilinx, Inc
+ 
+  Licensed under the Apache License, Version 2.0 (the "License"). You may
+  not use this file except in compliance with the License. A copy of the
+  License is located at
+ 
+      http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations
+  under the License.
+ */
+
+// ------ I N C L U D E   F I L E S -------------------------------------------
+// Local - Include Files
+#include "ReportAieShim.h"
+#include "core/common/query_requests.h"
+#include "core/common/device.h"
+#include <boost/optional/optional.hpp>
+#include <boost/property_tree/json_parser.hpp>
+
+#define fmt4(x) boost::format("    %-22s: " x "\n")

```

Comment:
Please remove the define statements and use boost::format's ability to insert these spaces.